<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>261</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AS600-mini TTF2CBitmap</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="fontPathLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>384</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="selectFontPushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>8</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Font</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputTextEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="lcdOutputPushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate LCD Output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printerOutputPushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Printer Output</string>
    </property>
   </widget>
   <widget class="QLabel" name="previewLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>96</width>
      <height>96</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="previewComboBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>41</y>
      <width>180</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="bitmapCharSizeLabel">
       <property name="text">
        <string>Bitmap Char Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="bitmapCharSizeWSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>96</number>
       </property>
       <property name="value">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bitmapCharSizeMultiplyLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="bitmapCharSizeHSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>96</number>
       </property>
       <property name="value">
        <number>16</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>83</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="charPaddingLabel">
       <property name="text">
        <string>Padding</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="charPaddingSpinBox">
       <property name="maximum">
        <number>96</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="colorToleranceLabel">
       <property name="text">
        <string>Tolerance</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="colorToleranceDoubleSpinBox">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.250000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>fontPathLineEdit</tabstop>
  <tabstop>selectFontPushButton</tabstop>
  <tabstop>bitmapCharSizeWSpinBox</tabstop>
  <tabstop>bitmapCharSizeHSpinBox</tabstop>
  <tabstop>charPaddingSpinBox</tabstop>
  <tabstop>colorToleranceDoubleSpinBox</tabstop>
  <tabstop>outputTextEdit</tabstop>
  <tabstop>lcdOutputPushButton</tabstop>
  <tabstop>printerOutputPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
